src/debugger/direct/directDebugAdapter.ts

```diff
@@ -156,6 +168,15 @@ export class DirectDebugAdapter extends ChromeDebugAdapter {
         super.disconnect(args);
     }
 
+    protected async onPaused(notification: Crdp.Debugger.PausedEvent, expectingStopReason = this._expectingStopReason): Promise<IOnPausedResult> {
+        // excluding Hermes native function calls from call stack, since VS Code can't process them properly (https://github.com/facebook/hermes/issues/168)
```

Comment:
```suggestion
        // excluding Hermes native function calls from call stack, since VS Code can't process them properly 
        // More info: https://github.com/facebook/hermes/issues/168
```